controller/index.js

```diff
@@ -1,18 +1,49 @@
 const db = require('../db/index.js');
+const exampleTicker = require('../sampledata/price.js');
 
+// generate data from req and body of req, see docs for right shape
+var Ticker = function(req) {
+  let tickerData = {};
+  tickerData.ticker = req.params.ticker;
+  tickerData.name = req.body.name ? req.body.name : null;
+  // QUESTION: for the prices do we need to convert the dates to a Date type to insert into collection?
+  tickerData.prices = req.body.prices ? req.body.prices : [];
+  return tickerData;
+}
+
+// QUESTION: why is there a next on these functions, but it's never called?
```

Comment:
sorry for the late comment! I think that was me from before ... (during the front end portion)